Tests/LanguageServerProtocolTests/LanguageServerProtocolTests.swift

```diff
@@ -33,4 +38,12 @@ final class LanguageServerProtocolTests: XCTestCase {
     XCTAssertEqual(encodedURI, fileURI)
     XCTAssertEqual(encodedURI.hashValue, fileURI.hashValue)
   }
+
+  func testFileChangeTypeEncoding() throws {
```

Comment:
Need to update linux tests for the addition

Reply:
Done